Cargo.toml

```diff
@@ -40,9 +40,29 @@ image = "0.25.1"
 reqwest = { version = "0.12.4", features = ["blocking"] }
 base64 = "0.22.1"
 
+# Config for 'cargo dist'
+[workspace.metadata.dist]
+# The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.16.0"
+# CI backends to support
+ci = "github"
+# The installers to generate for each app
+installers = []
```

Comment:
for ease of installing, you may want to add some installers :) this will let people install the pre-built binaries instead of having to compile on their machine with `cargo install`